playground/Stress/Stress.ApiService/Program.cs

```diff
@@ -55,6 +56,19 @@
     return "Counter incremented";
 });
 
+app.MapGet("/overflow-counter", (TestMetrics metrics) =>
+{
+    for (var i = 0; i < 100; i++)
+    {
+        for (int j = 0; j < 100; j++)
+        {
+            metrics.IncrementCounter(1, new TagList([new KeyValuePair<string, object?>($"add-tag-{i}", j.ToString(CultureInfo.InvariantCulture))]));
```

Comment:
// Emit measurements to ensure at least 2000 unique tag values are emitted,  
// matching the default cardinality limit in OpenTelemetry.  
for (var i = 0; i < 2005; i++)  
{  
    metrics.IncrementCounter(1, new TagList([new KeyValuePair<string, object?>("key", i.ToString(CultureInfo.InvariantCulture))]));  
}

^ This maybe simpler.... Irrespective, please do add a code-comment that 2000 is the default limit in OTel SDKs.